pages/blog/blog-post.js

```diff
@@ -0,0 +1,301 @@
+
+  @media(min-width:${mdBreakpoint}) {
+    p img {
+      margin: 0 -30px;
+      width: calc(100% + 60px);
+    }
+  }
+
+  @media(min-width:${lgBreakpoint}) {
+    p img {
+      margin: 0 -120px;
+      width: calc(100% + 240px);
+    }
+  }
+`
+
+const TeamMemberAuthor = styled(TeamMember)`
+  justify-content: flex-start;
+`
+
+const DateContainer = styled.div`
+  margin-bottom: 1.5rem;
+`
+
+const AuthorContainer = styled.div`
+  margin-top: 1.5rem;
+`
+
+const ShareContainer = styled.div`
+  margin-top: 1.5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+`
+
+const ShareButtonRow = styled.div`
+  display: flex;
+  flex-direction: row;
+
+  > * {
+    padding: 10px;
+  }
+
+  > :last-child {
+    margin-right: -10px;
+  }
+`
+
+const SocialLink = styled.div`
+  color: #444444;
+  opacity: 0.6;
+  cursor: pointer;
+
+  &:hover {
+    color: #444444 !important;
+    opacity: 1;
+  }
+`
+
+const HorizontalRule = styled.hr`
+  border-width: 2px;
+  border-color: ${props => props.theme.pineCone};
+  margin-top: 4rem;
+`
+
+const ButtonContainer = styled.div`
+ display: flex;
+ flex-direction: row;
+ justify-content: space-between;
+`
+
+const BlogPostContent = (props) => {
+  const {
+    title,
+    metaDescription,
+    date,
+    heroImage,
+    content,
+    authorTeamMember,
+    authorExternal,
+    url,
+  } = props
+
+  const author = authorTeamMember ?
+    (<TeamMemberAuthor
+      imageUrl={authorTeamMember.image.url}
+      title={authorTeamMember.name}
+      subtitle={authorTeamMember.responsibilityArea}
+      email={authorTeamMember.email}
+    />) :
+    <p>{authorExternal}</p>
+
+  const pageUrl = `https://www.masifunde.de/${url.asPath}`
+  const shareMessage = ''
+
+  return (
+    <div>
+      <Head title={title} description={metaDescription} />
+
+      {heroImage && <Hero
+        heroSize="small"
+        backgroundPositionX="70%"
+        imageUrl={heroImage.url}
+      />}
+
+      <PageSection>
+        <div className="offset-lg-2 col-lg-8">
+          <DateContainer>
+            <h5>{date}</h5>
+          </DateContainer>
+          <BlogTitle>{title}</BlogTitle>
+          <BlogMarkdown source={content} />
+          <div className="row">
+            <AuthorContainer className="col-6">
+              <h4>{props.authorText}</h4>
+              {author}
+            </AuthorContainer>
+            <ShareContainer className="col-6">
+              <h4>{props.shareText}</h4>
+
+              <ShareButtonRow>
+                <FacebookShareButton
+                  url={pageUrl}
+                  quote={shareMessage}
+                >
+                  <SocialLink>
+                    <FaFacebook size={24} />
+                  </SocialLink>
+                </FacebookShareButton>
+
+                <TwitterShareButton
+                  url={pageUrl}
+                  title={shareMessage}
+                >
+                  <SocialLink>
+                    <FaTwitter size={24} />
```

Comment:
This `24` is used twice, so we should turn it into a constant field in this file.